modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -9,6 +9,9 @@ The default installation of metering configures Hive to use an embedded Java dat
 
 There are 4 configuration options you can use to control the database used by Hive metastore: url, driver, username, and password.
 
+
+You will create your MySQL or Postgres instance with a username and secret. Then either in the Kubernetes CLI or by making a yaml file, you will make a secret. The secretName you use for this secret needs to map to you Metering CR `spec.hive.spec.config.db.secretName`.
```

Comment:
I think we may want to provide an example for how to create the secret by the CLI too:
```
kubectl -n $METERING_NAMESPACE create secret generic <1> --from-literal=username=<2> --from-literal=password=<3>

<1> The name of the secret
<2> The username of the database
<3> The password of the database.
```